compiler/noirc_frontend/src/tests.rs

```diff
@@ -537,9 +537,15 @@ mod test {
         ";
         let errors = get_program_errors(src);
         assert!(!has_parser_error(&errors));
-        assert!(errors.len() == 1, "Expected 1 error, got: {:?}", errors);
+        assert!(errors.len() == 2, "Expected 2 errors, got: {:?}", errors);
```

Comment:
why did this get changed?

Reply:
It should not have, git confused master tag vs master branch so I merged an out-of-date version. I removed the bad tag and fixed this.